b-real-time-htb.js

```diff
@@ -72,6 +72,13 @@ function BRealTimeHtb(configs) {
      * @private {object}
      */
     var __bidTransformers;
+    
+    /**
+     * Bidding endpoint
+     * 
+     * @private {string}
+     */
+    var __endpoint = 'ib.adnxs.com/ut/v2/prebid';
```

Comment:
Please add `//` in front of the endpoint url.

Reply:
In addition, please make the following changes to the mock library file for the unit test.
https://github.com/indexexchange/b-real-time-adapter/blob/bcae20c9e71263df5eee731240243fc507e7c745/spec/support/libraryStubData.js#L10-L26
line 12: `return 'http://';` = > `return 'http:';` (reason why '//' change is needed)
line 18: `return 'localhost';` => `return 'http://localhost';`
line 21: `return 'desktop';` => `return 'Mozilla/5.0 (Windows; U; Windows NT 6.1; rv:2.2) Gecko/20110201';` 

Thanks